### PR TITLE
8270925: replay dump using CICrashAt does not include inlining data

### PR DESCRIPTION
**Pull request by dean-long**
---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8270925](https://bugs.openjdk.java.net/browse/JDK-8270925): replay dump using CICrashAt does not include inlining data

### Reviewers
 * [Vladimir Kozlov](https://openjdk.java.net/census#kvn) (@vnkozlov - **Reviewer**)
 * [Tobias Hartmann](https://openjdk.java.net/census#thartmann) (@TobiHartmann - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/4908/head:pull/4908` \
`$ git checkout pull/4908`

Update a local copy of the PR: \
`$ git checkout pull/4908` \
`$ git pull https://git.openjdk.java.net/jdk pull/4908/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 4908`

View PR using the GUI difftool: \
`$ git pr show -t 4908`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/4908.diff">https://git.openjdk.java.net/jdk/pull/4908.diff</a>

</details>

### CODE REVIEW

**Review by vnkozlov — APPROVED** [review]
Good.

**Comment by dean-long** [comment]
Thanks Vladimir.

File: test/hotspot/jtreg/compiler/ciReplay/TestInlining.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2016, 2021, Oracle and/or its affiliates. All rights reserved.
```
Comment (TobiHartmann) [inline test/hotspot/jtreg/compiler/ciReplay/TestInlining.java:2]: I think the copyright date should just be 2021, right?

**Review by TobiHartmann — APPROVED** [review]
Looks good to me otherwise.

File: test/hotspot/jtreg/compiler/ciReplay/TestInlining.java (hunk 1 above)
Comment (dean-long) [reply test/hotspot/jtreg/compiler/ciReplay/TestInlining.java:2]: I copied TestVMNoCompLevel.java and then modified it, so I think keeping the start year is correct, as long as that content was non-trivial.  But I can change it if you feel strongly about it.

Comment (TobiHartmann) [reply test/hotspot/jtreg/compiler/ciReplay/TestInlining.java:2]: Okay, makes sense then to keep it as is.

**Comment by dean-long** [comment]
Thanks Tobias.

**Pull request closed without merging**